src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -34,12 +35,20 @@
      (str gcs-uri))))
 
 (defonce default-metadata-includes ["backendLogs" "backendStatus" "end" "executionStatus" "callCaching:hit" "failures"
-                                    "id" "inputs" "jobId" "outputs" "start" "status" "stderr" "stdout" "submission" "subworkflowId"
-                                    "submittedFiles:inputs" "workflowLog" "workflowName" "workflowRoot"])
+                                    "id" "jobId" "start" "status" "stderr" "stdout" "submission" "subworkflowId"
+                                    "workflowLog" "workflowName" "workflowRoot"])
 
 (defonce metadata-timing-includes ["attempt" "description" "end" "endTime" "executionEvents" "executionStatus"
                                    "shardIndex" "start" "startTime" "workflowName"])
 
+(defonce metadata-inputs-includes ["submittedFiles:inputs" "inputs"])
+
+;; requesting executionStatus for outputs makes the response payload somewhat larger, but it forces Cromwell to return
+;; data about every call in a task scatter. Otherwise, if we request just includeKey=outputs and a scatter contains both
+;; successes and failures, Cromwell will return only those calls that have outputs, i.e. the successes. Since our CLJS code
+;; identifies calls by their index in an array, this causes off-by-one (or off-by-many) functional errors.
```

Comment:
Good catch! *Ugh* about the parallel arrays. Without thinking about it too much more, I can imagine that "that's just the way it is" and there isn't a better way to handle this. Curious if there is a better solution that just isn't worth investing the energy to implement.

Reply:
you caught me - there is a potentially better way that I didn't want to implement - both b/c of development effort and risk of error. I *could* inspect the array for missing shards and insert them and/or use a function to extract a matching shard instead of relying on the array index. Both of these felt too easy to get wrong, so I felt this was overall the right solution.

Reply:
It works for me, especially since you've explicitly documented that array order dependency which someone might otherwise miss.